### DISCUSSION TITLE
v1.2.3 - Fix RTC Typo

### DISCUSSION THREAD

**Discussion by Wenn0101** [sparkfun/Arduino_Apollo3#405] [General]
Minor Updates:
Fix Typo in RTC Library

<hr /><em>This discussion was created from the release <a href='https://github.com/sparkfun/Arduino_Apollo3/releases/tag/v1.2.3'>v1.2.3 - Fix RTC Typo</a>.</em>

**Comment by adamgarbo** [comment]
Hi @Wenn0101,

Any chance we can get this release pushed out the door? There are no other outstanding issues that immediately come to mind. 

Cheers,
Adam

**Reply by Wenn0101** [reply]
Yep! I Just made it live!
